definitions/features/puppet_server.rb

```diff
@@ -56,6 +56,14 @@ def ca_directory_path
   end
 
   def puppet_ssldir_path
-    execute!('puppet master --configprint ssldir')
+    if puppet_version.major >= 6
```

Comment:
@patilsuraj767 ,
while testing this PR,  observed that `puppet master` command is deprecated in latest versions but `puppet config print` command available in all versions.
So instead of conditional run, can we just use `execute!("#{puppet_path} config print ssldir")`?

Reply:
Yes, agreed. 
I have made the required changes.